Runtime/Core/ProBuilderMeshFunction.cs

```diff
@@ -240,6 +240,19 @@ void GeometryWithPoints(Vector3[] points)
             Refresh();
         }
 
+        /// <summary>
+        /// Rebuilds the mesh with different vertices positions but same faces
+        /// </summary>
+        /// <param name="points">New vertices positions</param>
+        internal void ReplaceVertices(Vector3[] points)
```

Comment:
couldn't this just use the `positions` setter? i'm assuming that this is used just to handle resizing the bounds, in which case there isn't any need to rebuild the shared vertex cache or lookup.

likewise, no need for a complete mesh rebuild, you can just upload the positions directly to the mesh and avoid a lot of unnecessary overhead.